scripts/plots/cam/exevs_cam_nam_firewxnest_grid2obs_plots.sh

```diff
@@ -199,8 +199,10 @@ if [ -d $log_dir ]; then
 fi
         
 if [ $SENDCOM = "YES" ]; then
- mkdir -m 775 -p $COMOUTplots
- cpreq evs.plots.${MODELNAME}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
+ if [ -s evs.plots.${MODELNAME}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar ]; then
+  mkdir -m 775 -p $COMOUTplots
+  cpreq evs.plots.${MODELNAME}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
```

Comment:
@PerryShafran-NOAA Should this "cpreq" be "cp -v"?

@ShelleyMelchior-NOAA

Reply:
Whoops, did I not change that???

Reply:
@AliciaBentley-NOAA Fixed.  I thought I had that already updated but I guess I missed it.  Thanks for your eagle eyes check!